docs/guides/migrating-from-Camunda-Platform.md

```diff
@@ -0,0 +1,400 @@
+---
+id: migrating-from-Camunda-Platform
+title: Migrating from Camunda Platform
+---
+
+This guide describes how to migrate process solutions developed for Camunda Platform to run them on Camunda Cloud. You will learn about necessary steps, and also limitations of migration.
+
+Please note the following important considerations:
+
+1. **Migrating from Camunda Platform to Camunda Cloud is not a paved path**. As you will read below, there are several differences between the products and not all solutions are easy to migrate. This guide describes our experiences with customer projects so far, but is not a "turn-key"-migration procedure.
+
+2. **It is not necessarily recommended to migrate from Camunda Platform to Camunda Cloud**. Camunda Platform is a great product with existing support; there is no need to migrate existing solutions unless you have a good reason to (e.g. run it on SaaS, certain requirements on scalability or fault-tolerance, etc.) Therefore, we view migration more as an exception than a rule. Still, knowing the basic steps of migration can be helpful in building solutions with Camunda Platform, that will make migration easier in the future if necessary.
+
+Planning to migrate your project? [Talk to us](https://camunda.com/contact/) so we can help where needed.
```

Comment:
Do we want to route everyone through this Contact Us form? Or do we want to have options for the community forums and Slack as well? If so, we should link to this page - https://docs.camunda.io/contact

Reply:
I think you are right - linking that page makes more sense. Thx! I will take care of updating the PR

Reply:
https://github.com/berndruecker/camunda-cloud-documentation/commit/fc4bc097ce039eb722cbf6f328277bdd08119658